app/objects/c_operation.py

```diff
@@ -20,7 +19,7 @@
 
 
 class OperationSchema(ma.Schema):
-    id = ma.fields.Integer()
+    id = ma.fields.String()
```

Comment:
There is a `UUID` field type in marsmallow. Do we want to use that?

Reply:
We'd have to then parse ids as UUID when searching, right?

Reply:
What searching?

Reply:
`data_svc.locate()` or directly searching through the `ram`